# 4. Написать функцию которая принимает на вход число от 1 до 100. Если число равно 13, функция поднимает
# исключительную ситуации ValueError иначе возвращает введенное число, возведенное в квадрат. Далее написать основной
# код программы. Пользователь вводит число. Введенное число передаем параметром в написанную функцию и печатаем
# результат, который вернула функция. Обработать возможность возникновения исключительной ситуации,
# которая поднимается внутри функции.

def value_error_func(number):
    if number != 13:
        number = number ** 2
        return number
    else:
        raise ValueError('о, боже! ТРИНАДЦАТЬ')


user_number = int(input('Введите число от 1 до 100: '))

print('Квадрат введеного числа равен ', value_error_func(user_number))
